ansible/roles/common/tasks/main.yml

```diff
@@ -121,3 +121,7 @@
   authorized_key: key="{{ lookup('file', item) }}" user=deploy
   with_fileglob:
     - ssh_keys/*
+
+- name: enable systemctl services for deploy user
+  become: true
+  action: loginctl enable-linger deploy
```

Comment:
```suggestion
  action: loginctl enable-linger deploy
  become: true
```
To ensure the action is run as root user.

Reply:
Wait, the order matters?

Reply:
Missed the actual `become: true`. Please ignore this.